backend/src/common/util/pipelinerun.go

```diff
@@ -794,7 +794,7 @@ type PipelineRunInterface struct {
 func (pri *PipelineRunInterface) Create(ctx context.Context, execution ExecutionSpec, opts metav1.CreateOptions) (ExecutionSpec, error) {
 	pipelinerun, ok := execution.(*PipelineRun)
 	if !ok {
-		return nil, fmt.Errorf("execution is not a valid ExecutionSpec for Argo Workflow")
+		return nil, fmt.Errorf("execution is not a valid ExecutionSpec for Tekton")
```

Comment:
I think this likely needs to stay as Argo Workflow right? There are other instances of that in this file.

Reply:
This is what it's set at in upstream so this should be okay.